### ISSUE TITLE
query Cache Key Bug

### ISSUE THREAD

**Issue by jx-qyj** [zzzprojects/EntityFramework-Plus#5]
```C#
var db= DB.Create();
long a = 8;
var val1= db.bbs_topic.Where(e => e.sectionid==a).FromCache("bbs_topic").ToList().Count; //13
var val1_old = db.bbs_topic.Where(e => e.sectionid == a).ToList().Count; //13
a = 12;
var val2 = db.bbs_topic.Where(e => e.sectionid == a).FromCache("bbs_topic").ToList().Count; //13
var val2_old = db.bbs_topic.Where(e => e.sectionid == a).ToList().Count; //5
```

Why ‘val2’ is equal to 13? It should be 5.

**Comment by zzzprojects** [comment]
Thank you for reporting this issue.

We successfully reproduced and fixed it. We are currently adding more unit test to make sure this bug never happen again.

A new version should be available today or within the weekend.

**Comment by zzzprojects** [comment]
This issue is fixed in the v1.0.6.

The cache key now concatenates the query string and parameter to create a unique key.

Let us know if the issue is fixed for you.

**Comment by zzzprojects** [comment]
Closing issue due to no answer and bug fixed in unit tests
